modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-network-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network, `br-ex1`, that has a different configuration to the default `br-ex` bridge. You must define the additional bridge in the target interface definition of a machine config manifest file.  
```

Comment:
Let's add one small additional paragraph here:

~~~
This machine configuration will create a file at `/etc/ovnk/extra_bridge` containing the name of the interface intended to host this secondary bridge network.
~~~

Reply:
Updated to state:

> The Machine Config Operator uses the file to create a new file in the `/etc/ovnk/extra_bridge` directory. The new file includes the name of the bond interface that hosts the additional `br-ex` bridge network.